explorer/interpreter/resolve_names.cpp

```diff
@@ -33,12 +33,19 @@ static auto AddExposedNames(const Declaration& declaration,
       // Nothing to do here
       break;
     }
+    case DeclarationKind::DestructorDeclaration: {
```

Comment:
I don‘t know how to avoid them… Do you have a idea?

Reply:
Similar to `ImplDeclaration`, there should be nothing to do here. It should be skipped.

If this is hitting limits for your time, maybe add a TODO so that we don't forget this? Right now, it's actually going to be creating bugs, because someone could probably write code similar to: `var x: auto = Class.destructor`; that would succeed because of the `enclosing_scope.Add("destructor", ...)` name addition, and I don't think that code should work.